components/ping_ai_copilot/extension/content/content_scripts/content.js

```diff
@@ -0,0 +1,156 @@
+import "../ui/style.css"
```

Comment:
Is this import working? this is not a module to imported in a js file and might require to use styled components or to declare this as a static resource

Reply:
there is no need to import css in content scripts, its css file path is added in manifest itself. will remove this import